eth/ethconfig/config.go

```diff
@@ -67,7 +67,8 @@ var Defaults = Config{
 	TrieTimeout:             60 * time.Minute,
 	SnapshotCache:           102,
 	Miner: core.Config{
-		GasCeil:  8000000,
+		GasCeil:  18000000,
+		GasFloor: 1000000,
```

Comment:
What is this `GasFloor`? We can't guarantee we'll receive transactions to reach any floor

Reply:
These are default values, I agree, Floor should be set to the `MinGasLimit`. Any objections to setting it to the MinGasLimit ? @wizeguyy

Reply:
I removed the GasFloor here and will set it to minGas on the startup of the worker